elements/cloud-init-growpart-lvm/package-installs.yaml

```diff
@@ -0,0 +1,4 @@
+cloud-utils-growpart:
```

Comment:
Can we move this into pkg-map? Similar to https://github.com/openstack/diskimage-builder/blob/master/diskimage_builder/elements/grub2/pkg-map ?

Reply:
pkg-map didn't work for me, it copied the package-map file and ignored it afterwards with no error given